src/__tests__/http-test.ts

```diff
@@ -963,8 +957,8 @@ function runTests(server: Server) {
       expect(JSON.parse(response.text)).to.deep.equal({
         data: {
           uploadFile: {
-            originalname: 'http-test.js',
-            mimetype: 'application/javascript',
+            originalname: 'http-test.ts',
+            mimetype: 'video/mp2t',
```

Comment:
Don't really like this :(
Does multer supports explicitly specifying MIME type?

Reply:
I think it's just inferred from the extension. We could set up a file just for testing uploads (i.e. `fixtures/test.txt`).

Reply:
@danielrearden Please make a separate PR that attach with Blob as second argument:
https://developer.mozilla.org/en-US/docs/Web/API/Blob